File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        
        unordered_map<int,int>mp;

        for(int i=0 ; i<nums.size() ; i++){
            mp[nums[i]]+=1;
        }
int c=nums.size()/2;
        for(auto it:mp)
        {
            if(it.second>c) return it.first;
        }
        return 0;
    }
};

File: Data Structures & Algorithms/is-anagram/submission-0.java
class Solution {
    public boolean isAnagram(String s, String t) {
      int len1=s.length();
      int len2=t.length();

      if(len1!=len2) return false;
      int[] first=new int[26];
      int[] second=new int[26];

      for(int i=0;i<len1;i++){
         char ch1=s.charAt(i);
         char ch2=t.charAt(i);
         first[ch1-'a']+=1;
         second[ch2-'a']+=1;
      }

      for(int i=0;i<26;i++){
        if(first[i]!=second[i]) return false;
      }
      return true;


    }
}
